core/test/e2e/api/v2/content/utils.js

```diff
@@ -0,0 +1,11 @@
+const url = require('url');
+const _ = require('lodash');
+const API_URL = '/ghost/api/v2/content/';
+
+module.exports = {
+    API: {
+        getApiQuery(route) {
+            return url.resolve(API_URL, route);
+        }
+    }
+};
```

Comment:
I think this and `core/test/e2e/api/v2/admin/utils.js` can be combined and moved to `core/test/e2e/api/v2/shared/utils.js`, and API_URL can be configured with functional param. No need to break it to `admin` and `content` for this, wdyt?

Reply:
Hm but then you have to call `localUtils.API.queryApi(content:true)`

Reply:
My thinking was that the query structure is always the same and the url get's generated under the hood automatically

Reply:
You mean specs can just directly import utils and call `getAPIQuery` and not worry about content/admin as its under the hood? Yeah otherwise will have to pass explicit type, like `localUtils.API. getApiQuery(route, type="content|admin")`. I see the point in keeping this, I was thinking about code duplication.